### ISSUE TITLE
Member - Green highlight on member jumps to other member when selecting member

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#25884]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
**Issue found when executing PR** https://github.com/Expensify/App/pull/22622
## Action Performed:
1. Go to staging.new.expensify,com
2. Go to workspace settings with some members
3. Go to Members
4. Select any member

## Expected Result:
There will be no issue when member is selected

## Actual Result:
The green highlight on the member row jumps to other member row when the member is selected, which can be confusing

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.57-0

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/a7c7fafd-50ef-4d65-88f8-c88676b076a9

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by ishpaul777** [comment]
passing a ref seems to Pressable with feedback seems to resolve the isssue

**Comment by Santhosh-Sellavel** [comment]
@deetergp 
This is not a deploy blocker, I guess this is expected cc: @cristipaval @thiagobrez. 

If this needs changed can be handled along with phase 3 of SelectionList Improvement!
cc: @shawnborton

**Comment by ishpaul777** [comment]
I agree with @getusha this behaviour feels confusing sometimes for eg. user is checking off backward with arrowkeys

https://github.com/Expensify/App/assets/104348397/fd86e8de-e1c0-4c2c-b0dd-621ea8c00a37

**Comment by shawnborton** [comment]
Hmm I agree, that is weird and feels unexpected.

**Comment by thiagobrez** [comment]
> This is not a deploy blocker, I guess this is expected
> If this needs changed can be handled along with phase 3 of SelectionList Improvement!

Agree with @Santhosh-Sellavel here. We can handle it on https://github.com/Expensify/App/issues/20354

@shawnborton So I guess we should keep the highlight in the same item when selecting it?

Here's a relevant comment regarding the highlight in Workspace Members vs. Workspace Invite members: https://github.com/Expensify/App/pull/22622#issuecomment-1682029337

**Comment by mountiny** [comment]
Coming from here, removing the deploy blocker as this was planned behaviour with the new list but its a new concept which feels unexpected. Would be great to discuss with the design team what we should do about this https://expensify.slack.com/archives/C01GTK53T8Q/p1692963954482309?thread_ts=1692952886.670059&cid=C01GTK53T8Q

**Comment by shawnborton** [comment]
> So I guess we should keep the highlight in the same item when selecting it?

That would be my vote, yup!

**Comment by cristipaval** [comment]
Holding it on https://github.com/Expensify/App/issues/20354 as per [this comment](https://github.com/Expensify/App/issues/25884#issuecomment-1692896677)

**Comment by getusha** [comment]
@samh-nl your proposal is duplicate of mine
https://github.com/Expensify/App/issues/25884#issuecomment-1692356398
And sharing a diff is against the contribution policy. 🙂

**Comment by cristipaval** [comment]
Still on hold. Marking this `Weekly`

**Comment by thiagobrez** [comment]
PR is ready to review here: https://github.com/Expensify/App/pull/27246

**Comment by JmillsExpensify** [comment]
$500 payment approved for @allroundexperts based on BZ summary.
